Sources/SnapshotTesting/Strategy/NSView.swift

```diff
@@ -8,17 +8,23 @@ extension Strategy {
   }
 
   public static func view(precision: Float) -> Strategy<NSView, NSImage> {
-    return Strategy.image(precision: precision).contramap {
-      precondition(!($0 is WKWebView), """
```

Comment:
The overload made error messages really bad. So instead I've decided to bake the logic in here.

Reply:
Basically, if you passed an `A` that doesn't have a default strategy to `assertSnapshot(matching:)` without naming an explicit strategy, the compiler assumed the overload instead, producing this error:

> 🛑 Cannot convert value of type 'A' to expected argument type 'WKWebView'